gpm_api/tests/io/test_disk.py

```diff
@@ -0,0 +1,2 @@
+def test_get_dist_daily_filepaths_private() -> None:
```

Comment:
Here I see that you didn't test `find_filepaths`. 
I think we could create a GPM `base_dir` in the test directory (or create a temporary directory when executing the test)
with the expected structure `<tests/data/GPM>/<product_type>/<version>/<product_category>/<product>/<year>/<month>/<day>`,
then add some dummy files (with realistic names) and test that `find_filepaths` returns their list. 
Alternatively (to be discussed with @sphamba), we could create such a directory in `test/data/raw/GPM`, write there the test HDF files to test the data reading ... and then have a `test/data/expected/GPM` directory with the expected gpm_api xr.Dataset (saved in netCDF or Zarr format).